builder/vsphere/common/step_reattach_cdrom_test.go

```diff
@@ -43,23 +43,68 @@ func TestStepReattachCDRom_Run(t *testing.T) {
 			},
 			expectedVmMock: &driver.VirtualMachineMock{
 				EjectCdromsCalled:        true,
-				RemoveCdromsCalled:       true,
 				ReattachCDRomsCalled:     true,
 				FindSATAControllerCalled: true,
-				AddCdromCalledTimes:      8,
+				AddCdromCalledTimes:      6,
 				AddCdromTypes: []string{
 					"sata", "sata",
 					"sata", "sata",
 					"sata", "sata",
-					"sata", "sata",
 				},
-				AddCdromPaths: []string{
-					"[datastore] /iso/linux.iso", "[datastore] /iso/linux.iso",
-					"[datastore] /iso/linux.iso", "[datastore] /iso/linux.iso",
-					"[datastore] /iso/linux.iso", "[datastore] /iso/linux.iso",
-					"[datastore] /iso/linux.iso", "[datastore] /iso/linux.iso",
+			},
+			fail: false,
+		},
+		{
+			name: "Reattach 2 less CD-ROM devices than initially added",
```

Comment:
```suggestion
			name: "Successfully attach 2 fewer CD-ROM devices than initially added.",
```

Reply:
Done! I removed a dot at the end because from looking at other tests I see the sentence ends with no dot at the end. Please tell if you want me to add it.

Reply:
Resolved in 860f7101405d68474e1acadf652ef640f3e43b2b.